.flake8

```diff
@@ -1,3 +1,3 @@
 [flake8]
 max-line-length=120
-ignore: E301, E302, E305, E401, F401, W503, W504, W605, E252, E402, E501, E265
+ignore: E301, E302, E305, E401, F401, W503, W504, W605, E252, E402, E501, E265, W291
```

Comment:
Please revert, we don't want trailing whitespace.

Reply:
It's required to make bullet points show up correctly in markdown.

Reply:
In my testing, trailing whitespace is _not_ needed for bullet points. Can you provide an example?

Reply:
## Assumptions
    * You are not trying to overwrite a data table that already exists  
    * Your files follow a naming convention either like this...  
    NWD119844.CRAM  
    NWD119844.CRAM.CRAI  
    ...or this:  
    NWD119844.CRAM  
    NWD119844.CRAI  

^ When I originally put this in without the trailing spaces it currently has, all of it was on the same line. Newlines don't really consistently work in markdown unless there's two of them, which looks weird when you're dealing with bulletpoints.

Reply:
<br> is supposed on Jupyter, swapping to that instead